### PR TITLE
Log rally points to dataflash on startup

### PR DESCRIPTION
**Pull request by rmackay9**
This logs rally points to the dataflash on startup.  It doesn't attempt to log any changes to rally points although that's not too difficult a step after this PR is in.

### CODE REVIEW

File: libraries/DataFlash/LogStructure.h [hunk 1]
```diff
@@ -821,7 +831,7 @@ Format characters in the format string for binary log messages
     { LOG_GPS_RAWS_MSG, sizeof(log_GPS_RAWS), \
       "GRXS", "QddfBBBHBBBBB", "TimeUS,prMes,cpMes,doMes,gnss,sv,freq,lock,cno,prD,cpD,doD,trk" }, \
     { LOG_GPS_SBF_EVENT_MSG, sizeof(log_GPS_SBF_EVENT), \
-      "SBFE", "QIHBBdddfffff", "TimeUS,TOW,WN,Mode,Err,Lat,Long,Height,Undul,Vn,Ve,Vu,COG" }, \
+      "SBFE", "QIHBBdddfffff", "TimeUS,TOW,WN,Mode,Err,Lat,Lng,Height,Undul,Vn,Ve,Vu,COG" }, \
```
Comment (magicrub) [inline libraries/DataFlash/LogStructure.h]: Sneaky sneaky!

Comment (rmackay9) [inline libraries/DataFlash/LogStructure.h]: Ah darn!  Forgot I made that change.  I'll put that in a different commit.

**Comment by tridge** [comment]
looks good to me

**Comment by rmackay9** [comment]
Thanks, merged!

**Pull request merged**